src/Controller/Api/GroupController.php

```diff
@@ -62,9 +64,14 @@ public function __construct(GroupManagerInterface $groupManager, FormFactoryInte
     /**
      * Returns a paginated list of groups.
      *
-     * @ApiDoc(
-     *  resource=true,
-     *  output={"class"="Sonata\DatagridBundle\Pager\PagerInterface", "groups"={"sonata_api_read"}}
+     * @Operation(
```

Comment:
```suggestion
     * @SWG\Operation(
```
Can we use `@SWG` instead add another FQNC?

Reply:
I guess we could, but I'm not sure we should. Regardless the `SWG` alias was already imported, IMO we should avoid to propagate the usage of classes which are provided by packages which are not declared explicitly as direct dependencies.

nelmio/NelmioApiDocBundle#973.